set.mm

```diff
@@ -41653,9 +41709,10 @@ fewer connectives (but probably less intuitive to understand).
        class abstraction.  (Contributed by Mario Carneiro, 8-Sep-2013.) $)
     dfif6 $p |- if ( ph , A , B ) =
                  ( { x e. A | ph } u. { x e. B | -. ph } ) $=
-      ( cv wcel wa cab wn cun wo crab cif unab df-rab uneq12i df-if 3eqtr4ri )
-      BEZCFAGZBHZSDFAIZGZBHZJTUCKBHABCLZUBBDLZJACDMTUCBNUEUAUFUDABCOUBBDOPABCDQ
-      R $.
+      ( vy cv wcel wa cab wn cun wo crab cif eleq1w anbi1d unabw df-rab uneq12i
+      weq df-if 3eqtr4ri ) BFZCGZAHZBIZUCDGZAJZHZBIZKEFZCGZAHZUKDGZUHHZLEIABCMZ
+      UHBDMZKACDNUEUIUMUOBEBETZUDULABECOPURUGUNUHBEDOPQUPUFUQUJABCRUHBDRSAECDUA
+      UB $.
```

Comment:
I think a "$( $j usage 'dif6' avoids ... " should be added here, at least, to justify why this proof becomes longer (also for ~dfif3 and ~unopab)

Reply:
Done